WordPress/src/main/java/org/wordpress/android/ui/bloggingreminders/BloggingRemindersItem.kt

```diff
@@ -44,5 +52,11 @@ sealed class BloggingRemindersItem(val type: Type) {
         data class DayItem(val text: UiString, val isSelected: Boolean, val onClick: ListItemInteraction)
     }
 
+    data class TimeItem(
+        val time: UiString,
+        val isInvisible: Boolean = false,
```

Comment:
It looks like `isInvisible` is not being used anywhere?

Reply:
Need to seek clarification on this one.  We had discussion if you remember, whether the time selection should always be displayed or only after a day selection is made.  If we decide to default to display always, then this can be removed

Reply:
Update it now, to be visible only when at least a day is selected